metcdv3/coordinator.go

```diff
@@ -279,9 +279,10 @@ func (ec *EtcdV3Coordinator) Release(task metafora.Task) {
 		Commit()
 	if err != nil {
 		metafora.Warnf("Error releasing task %s while stopping: %v", tid, err)
+		return
 	}
-	if txnRes == nil || !txnRes.Succeeded {
-		metafora.Warnf("Failed to release task %s while stopping.", tid)
+	if !txnRes.Succeeded {
+		metafora.Warnf("Failed to release task %s while stopping: %+v", tid, txnRes)
 	}
```

Comment:
This is the important change: I keep seeing these error when my works shut down (e.g., interruption) and I don't know why they're failing. Now I will 😁